lib/pengine/bundle.c

```diff
@@ -176,11 +177,8 @@ create_docker_resource(pe_resource_t *parent, pe__bundle_variant_data_t *data,
                        pe__bundle_replica_t *replica,
                        pe_working_set_t *data_set)
 {
-        int offset = 0, max = 4096;
-        char *buffer = calloc(1, max+1);
-
-        int doffset = 0, dmax = 1024;
-        char *dbuffer = calloc(1, dmax+1);
+        GString *buffer = g_string_sized_new(4096);
+        GString *dbuffer = g_string_sized_new(1024);
```

Comment:
These constants are used several places.  I can't imagine them changing in the future, but it might be nice to be ready just in case.

Reply:
Yeah, I don't have a strong preference. I had asked Ken about defining constants for GString sizes and he said they might vary from place to place and no reason to shoehorn all the calls into using one size.

You could make a decent case for defining constants per-file (rather than in a header). I don't expect it to make much difference and (like you) I don't expect to change them. We're unlikely to downsize them, and the nice thing about GStrings is that if they're too small, nothing catastrophic happens.